examples/cdk-examples-java/pom.xml.t.js

```diff
@@ -6,7 +6,7 @@ const mavenFromNpm = name => ({
     repo: path.join(path.dirname(require.resolve(name)), 'maven-repo'),
 });
 
-const cdk = mavenFromNpm('aws-cdk-java');
+const cdk = mavenFromNpm('aws-cdk');
 const jsii = mavenFromNpm('jsii-java-runtime');
```

Comment:
I am wondering if it will make more sense to just use the published maven artifacts instead of generate the pom so this is iidiomatic

Reply:
That's basically what I'm doing here... Just figuring out the latest version numbers... Honestly this is just what I needed to fix in order to validate the maven staging artifacts... Just wanted those changes not to be lost :)

Reply:
We should move those examples to a separate repo so people can clone it and use as a basis for their app.